runbot_buildout/migrations/11.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
No coding here ;-)

Reply:
thanks. I'm stilly busy making tests work on runbot, so this PR might get a little bit noisy